content/en/storage-providers/curio/setup.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Setup Curio"
+description: "This guide will show you setup a new Curio cluster or migrate to Curio from Lotus-Miner"
+lead: "This guide will show you setup a new Curio cluster or migrate to Curio from Lotus-Miner"
+draft: false
+menu:
+    storage-providers:
+        parent: "curio"
+weight: 130
+toc: true
+---
```

Comment:
This should have aliases to the old lotus-provider page so it keeps current links intact.

Reply:
Alias are required to avoid broken links in large docs. There would be no need here as these docs were self contained. I see no link to any of these pages.